torch/onnx/symbolic_opset10.py

```diff
@@ -178,3 +179,67 @@ def flip(g, input, dims):
 
 def fmod(g, input, other):
     return g.op("Mod", input, other, fmod_i=1)
+
+
+@parse_args('v', 'v', 'v', 'i', 'i', 'i', 'v', 'i')
+def embedding_bag(g,
+                  embedding_matrix,
+                  indices,
+                  offsets,
+                  scale_grad_by_freq,
+                  mode,
+                  sparse,
+                  per_sample_weights,
+                  include_last_offset):
+    from torch.onnx.symbolic_opset9 import size, div
+
+    if not sym_help._is_none(per_sample_weights):
```

Comment:
Add similar check for `include_last_offset`.

`scale_grad_by_freq` is for training, maybe add check as well if exporting in training mode?